library/core/src/mem/mod.rs

```diff
@@ -627,12 +627,22 @@ pub unsafe fn zeroed<T>() -> T {
     }
 }
 
-/// Bypasses Rust's normal memory-initialization checks by pretending to
-/// produce a value of type `T`, while doing nothing at all.
-///
-/// **This function is deprecated.** Use [`MaybeUninit<T>`] instead.
-///
-/// The reason for deprecation is that the function basically cannot be used
+/// **This function is deprecated with extreme prejudice.**
+/// It is replaced by [`MaybeUninit<T>`], which must be used instead of this function.
```

Comment:
Have you looked at how this renders in rustdoc? The standard is one summary sentence, trying to keep it succinct so it renders on, in this case, the `std::mem` page. The previous one was already a teeny bit long.